pkg/dmqnode-common.pkg

```diff
@@ -0,0 +1,12 @@
+import defaults
```

Comment:
It looks like this might be in the wrong commit.

Reply:
Updated, it should be in the right commit now.